README.md

```diff
@@ -121,6 +123,10 @@ In this example, we require `should` for assertions, this helper module, as well
 
 We then have a set of mocha unit tests.  These tests check that the node loads correctly, and ensures it makes the payload string lower case as expected.
 
+## Initializing helper
+
+To get started, we need to tell the helper where to find the node-red runtime.  this is done by calling `helper.init(require.resolve('node-red'))` as shown.
```

Comment:
capitalization 😜

Reply:
I also wonder why we're doing this.  Can't we just `require('node-red')` then pass *that* to `helper.init()`?  Could that defeat the purpose of `helper.init()` entirely?

Reply:
In other words: can we not reach all of the objects we need from the thing returned by `require('node-red')`?